controllers/v1/projectAttributes.js

```diff
@@ -0,0 +1,149 @@
+/**
+ * name : projectAttributes.js
+ * author : PraveenDass
+ * created-date : 25-Nov-2024
+ * Description :  projectAttributes Controller.
+ */
+
+// dependencies
+let projectAttributesHelper = require(MODULES_BASE_PATH + '/projectAttributes/helper')
+
+/**
+ * projectAttributes service.
+ * @class
+ */
+module.exports = class ProjectAttributes extends Abstract {
+	constructor() {
+		super('project-attributes')
+	}
+	/**
```

Comment:
@praveenKDass can you please check the alignment of api doc is correct or not

Reply:
Resolved Alignment @VISHNUDAS-tunerlabs